examples/gridding/simple/toygrid.py

```diff
@@ -174,4 +188,4 @@ def create_parser():
 
         # Produce visibilities for this chunk of UVW coordinates
         vis = degrid(dirty, uvw, natural_weight, wavelength,
```

Comment:
Glad to see weights in both the gridder and degridder but need to confirm this is doing what I hope it's doing. Weights are usually only applied during gridding but this breaks the self-adjointness of the operator. Better to apply the sqrt of the weights going both ways. Is this what's happening here?